.github/workflows/tag-app-release.yml

```diff
@@ -0,0 +1,123 @@
+name: Tag app release
+
+on:
+  workflow_dispatch:
+    inputs:
+      app:
+        type: string
+        required: true
+        description: Application to tag for. Either `ingestion_server`, `api`, or `frontend`.
```

Comment:
Could this potentially be of type `choice`? [Example](https://github.blog/changelog/2021-11-10-github-actions-input-types-for-manual-workflows/). That way we wouldn't have to let this be a free-form field and perform the validation step below explicitly.

Reply:
Yes, definitely. I totally overlooked that.